packages/datadog-instrumentations/src/ldapjs.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
+
+const {
+  channel,
+  addHook,
+  AsyncResource
+} = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+
+function isString (value) {
+  return typeof value === 'string' || value instanceof String
+}
+
+function getCallbackArgIndex (args) {
+  let callbackIndex = -1
+  for (let i = args.length - 1; i >= 0; i--) {
+    if (typeof args[i] === 'function') {
+      callbackIndex = i
+      break
+    }
+  }
+  return callbackIndex
+}
+
+addHook({ name: 'ldapjs', versions: ['>=2'] }, ldapjs => {
+  const ldapSearchCh = channel('datadog:ldapjs:client:search')
+
+  shimmer.wrap(ldapjs.Client.prototype, 'search', search => function (base, options) {
+    if (ldapSearchCh.hasSubscribers) {
+      let filter = null
```

Comment:
Any reason to initialize to `null` instead of maintain as `undefined`?
```suggestion
      let filter
```